starter-code/css/main.css

```diff
@@ -1,8 +1,90 @@
-/*
-Colors:
+/*Colors:
 
 Text: 1A1A1A
 Green: #00B172
 White: #FFF
-
 */
+
+
+.container {
+  border: 5px solid red;
+}
+
+ nav {
+    border: 5px solid yellow;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header {
+  background-image: url(../images/landing.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: top center;
+  padding: 35px;
+}
```

Comment:
You can set a height to your header, for example 1000px to see more of the image